cdap-ui/app/directives/dag-plus/services/stores/nodes-store.js

```diff
@@ -130,7 +130,7 @@ class DAGPlusPlusNodesStore {
       nodeConfig.name = nodeConfig.plugin.label + '-' + this.uuid.v4();
     }
     if (!nodeConfig.id) {
-      nodeConfig.id = nodeConfig.plugin.label.replace(/[ \/]/g, '-');
+      nodeConfig.id = nodeConfig.plugin.label.replace(/[ \/]/g, '-') + '-' + this.uuid.v4();
```

Comment:
Does this still need the hyphen-replacement fix?

Reply:
This is done only on id. This shouldn't affect the pipeline. This is needed only for multiselect.